Vision/ImageSimilarity/README.md

```diff
@@ -0,0 +1,249 @@
+# Image Similarity Power Skill #
+
+This Power Skill uses the pre-trained [ResNet](https://keras.io/api/applications/resnet/)
+deep learning model to extract visual features from an image and to find the top n most similar
+images. 
+
+This skill is ideal for:
+
+1) Exploring your data to identify similar images during your data exploration phase
+2) Using in conjunction with Custom Vision Classification to further classify your data.
+3) For use in recommender type systems where similar images are of interest 
+4) Working with video where individual keyframes are extracted for analysis
+
+See the data folder for sample images used for in the skill. This uses the [Stanford Dogs dataset](http://vision.stanford.edu/aditya86/ImageNetDogs/)
+
+## Requirements
+
+In addition to the common requirements described in the root [README.md](https://github.com/Azure-Samples/azure-search-power-skills/blob/master/README.md) 
+file, this Power Skill requires access to a Custom Vision resource. This process will use object detection to illustrate 
+the captions and text extracted and its relevance to the topn similar images.
+
+To run this PowerSkill you will need:
+* docker
+* An Azure Blob storage container
+* A provisioned Azure Cognitive Search (ACS) instance 
+* A provisioned Azure Container Registry
+* A Cognitive Services key in the region you deploy ACS to
+
+Below is a full working example that you can get working end to end on sample data.
+
+## High level Process
+
+![image similarity process](images/image_similarity_flow.png)
+
+## How to implement
+
+This section describes how to get this working on sample data
+and how it can be amended for your data.
+ 
+1) ###Data
```

Comment:
The title markers `###` don't seem to work with numbered lists, at least on GH:

![image](https://user-images.githubusercontent.com/1165609/110026339-5d52e200-7ce5-11eb-8f89-1c73be04c5ef.png)

Reply:
@bleroy - Fixed the markdown